components/GraphDiagram/graph_util.ts

```diff
@@ -218,7 +218,9 @@ export function composeDOT({
       fontsize *= Math.max(1, Math.log10(unpackedSize) - 2);
     }
 
-    const vs = { root: level == 0, fontsize };
+    const link = new URL(location.origin);
+    link.searchParams.append('q', module.key);
+    const vs = { root: level == 0, fontsize, href: link.href };
```

Comment:
This adds links to the graph itself instead of having to parse the clicked module from the graph and then using `window.open()`

Reply:
It's `href` https://graphviz.org/docs/attrs/href/